src/docs/resources/faq.md

```diff
@@ -281,7 +281,7 @@ and met all our requirements and criteria.
 
 Dart runtimes and compilers support the combination of two critical features for
 Flutter: a JIT-based fast development cycle that allows for shape changing and
-stateful hot reloads in a language with types, plus an Ahead-of-Time compiler
+Stateful Hot Reloads in a language with types, plus an Ahead-of-Time compiler
```

Comment:
This is definitely just using English, not hyping a feature. I'd leave this lower case.

Reply:
Agreed that this isn't meant to reference the official feature, but "stateful" is still needed as a descriptor in this context. Will leave it as is.

Reply:
Right, it's fine to leave "stateful". Just because those three words are used together does not mean that it should be automatically made upper case. It has more to do with referring to the Official Feature (TM), or just describing.